src/librustc_metadata/rmeta/decoder/cstore_impl.rs

```diff
@@ -529,6 +528,6 @@ impl CrateStore for cstore::CStore {
 
     fn metadata_encoding_version(&self) -> &[u8]
     {
-        schema::METADATA_HEADER
+        rmeta::METADATA_HEADER
```

Comment:
Heh I just noticed the name of the method is kind of out of sync with what it's actually returning. No need to change it now though.